.drone.yml

```diff
@@ -462,7 +462,7 @@ steps:
     - mkdir build && cd build
     - cmake $CMAKE_FLAGS ..
     - make -j
-    - ASAN_OPTIONS="detect_leaks=0" ctest -j --output-on-failure
+    - ASAN_OPTIONS="detect_leaks=0" ctest -j4 --output-on-failure -E checkperf
```

Comment:
Why would we ever want to run checkperf on sanitized code?

Reply:
@lemire -E is exclude :)

Reply:
Ok. Then I am fine.